txn_writer.go

```diff
@@ -21,15 +21,43 @@ import (
 // Example string to be written to the manifest file
 // if no dependencies are found in the project
 // during `dep init`
-const exampleToml = `
-# Example:
+const exampleTOML = `
+## EXAMPLES & DOCS - safe to delete!
+#
+## Dependencies define constraints on how dependent projects should be
+## incorporated into Gopkg.lock. They are respected by dep whether
+## this project is the current project, or if it's a dependency.
 # [[dependencies]]
+## Required: the root import path of the project being constrained
+# name = "github.com/user/project"
+## Optional: an alternate location (URL or import path) for the project's source
 # source = "https://github.com/myfork/package.git"
+## Optional, but recommended: the version constraint to enforce for the project.
+## Only one of "branch", "version" or "revision" can be specified.
+# version = "1.0.0"
 # branch = "master"
-# name = "github.com/vendor/package"
-# Note: revision will depend on your repository type, i.e git, svc, bzr etc...
+## Note: revision will depend on your repository type; git and hg have SHA1s,
+## bzr a 3-part id, svn a revision number.
 # revision = "abc123"
-# version = "1.0.0"
+#
+## [[overrides]] follow the exact same structure as [[dependencies]], but supercede
```

Comment:
yeah, restructuring this. I dropped the section overall because it's duplicative of `[[dependencies]]`, but i'm re-adding it because the comment text should actually be different.